src/Zope2/utilities/copyzopeskel.py

```diff
@@ -182,7 +182,7 @@ def copyskel(sourcedir, targetdir, uid, gid, **replacements):
                 copydir(root, dirs, files, targetdir, replacements, uid, gid)
         finally:
             os.chdir(pwd)
-    except (IOError, OSError) as msg:
+    except OSError as msg:
```

Comment:
Cool, I had to look up that `IOError` has been merged into `OSError`, see https://docs.python.org/3/library/exceptions.html#OSError